cfme/tests/infrastructure/test_kubevirt.py

```diff
@@ -3,23 +3,94 @@
 import pytest
 
 from cfme.infrastructure.provider.kubevirt import KubeVirtProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
 from cfme.utils.update import update
 
+
 pytestmark = [
-    pytest.mark.provider([KubeVirtProvider], scope="module")
+    pytest.mark.provider([KubeVirtProvider]),
+    pytest.mark.usefixtures('setup_provider')
 ]
 
 
-def test_k6t_provider_crud(provider):
-    """Tests provisioning via PXE
+@pytest.fixture
+def vm_name():
+    vm_name = random_vm_name('k6tvm')
+    return vm_name
+
+
+@pytest.fixture
+def prov_data(vm_name):
+    return {'catalog': {'vm_name': vm_name}}
 
-    Metadata:
-        test_flag: v2v
-    """
-    provider.create()
+
+@pytest.fixture
+def temp_vm(appliance, provider, vm_name, prov_data, provisioning):
+    template_name = provisioning['template']
+    vm = appliance.collections.infra_vms.instantiate(name=vm_name,
+                                                     provider=provider,
+                                                     template_name=template_name)
+    note = ('template {} to vm {} on provider {}'.format(template_name, vm_name, provider.key))
+    prov_data.update({
+        'request': {
+            'email': 'template_provisioner@example.com',
+            'first_name': 'Template',
+            'last_name': 'Provisioner',
+            'notes': note}})
+    view = navigate_to(vm, 'Provision')
+    view.form.fill_with(prov_data, on_change=view.form.submit_button)
+    view.flash.assert_no_error()
+    vm.wait_to_appear()
+    yield vm
+    # vm.retire()
```

Comment:
Is there no delete method? Is there a BZ associated with kernel panic? Is this related to a specific kubevirt version, or deployment? Need some more detail, and a BZ skip if possible :)

Reply:
This was solved but forgot to remove the comment, retirement works now, changed it